modules/imgproc/include/opencv2/imgproc.hpp

```diff
@@ -3687,7 +3687,7 @@ data type.
 is \f$W \times H\f$ and templ is \f$w \times h\f$ , then result is \f$(W-w+1) \times (H-h+1)\f$ .
 @param method Parameter specifying the comparison method, see cv::TemplateMatchModes
 @param mask Mask of searched template. It must have the same datatype and size with templ. It is
-not set by default.
+not set by default. Currently, only the CV_TM_SQDIFF and CV_TM_CCORR_NORMED methods are supported.
```

Comment:
Constants with `CV_` prefix are deprecated and shouldn't be mentioned in docs. You could use `TM_SQDIFF` and `CV_TM_CCORR_NORMED` from [here](https://github.com/saskatchewancatch/opencv/blob/master/modules/imgproc/include/opencv2/imgproc.hpp#L3658).